wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
-Why am I banned?
-----------------
+# Handling Cheating or Foul Play
```

Comment:
the title should be the folder name (otherwise, update the folder name to match the title)

Reply:
Sticking to the folder name then. Maybe add a shortcut for `Foul Play`